cmd/nri-prometheus/main.go

```diff
@@ -14,5 +14,8 @@ func main() {
 		logrus.WithError(err).Fatal("while loading configuration")
 	}
 
-	scraper.Run(cfg)
+	err = scraper.Run(cfg)
+	if err != nil {
+		logrus.WithError(err).Fatal("execution finished with error")
```

Comment:
small thing, but finished indicates that it ran till completion, while it could not have been working at all

```suggestion
		logrus.WithError(err).Fatal("error occurred while running scraper")
```

or something like this